metricbeat/modules.d/system.yml

```diff
@@ -1,14 +1,20 @@
 - module: system
+  period: 10s
+  metricsets:
+    - cpu
+    - load
```

Comment:
`load` doesn't exist for Windows. Does `load` get deleted during packaging for Windows (check `before-build` in the Makefile)?

Reply:
It shoulds, but it's a good point, I must test this